CMakeLists.txt

```diff
@@ -99,7 +99,7 @@ else()
     set(WINE_CMD)
 endif()
 
-list(APPEND CMAKE_PREFIX_PATH /opt/rocm /opt/rocm/hcc /opt/rocm/hip)
+list(APPEND CMAKE_PREFIX_PATH /opt/rocm /opt/rocm/llvm /opt/rocm/hcc /opt/rocm/hip)
```

Comment:
Is it necessary to add /opt/rocm/llvm to `CMAKE_PREFIX_PATH`?

Reply:
It was not picking up clang++ over hcc even though I asked cmake to look for clang++ first. 
Though now it is making our HCC builds fail because it is picking up the system installed clang++.
http://micimaster.amd.com/blue/organizations/jenkins/MLLibs%2FMIOpen/detail/lookForClang/2/pipeline
I can remove it, but do you have suggestions on making sure the path gets picked up?